frontend/www/js/omegaup/components/problem/FilterTags.vue

```diff
@@ -28,31 +28,36 @@
 <script lang="ts">
 import { Vue, Component, Prop, Watch } from 'vue-property-decorator';
 import T from '../../lang';
+import { types } from '../../api_types';
 import VueTypeaheadBootstrap from 'vue-typeahead-bootstrap';
 @Component({
   components: {
     'vue-typeahead-bootstrap': VueTypeaheadBootstrap,
   },
 })
 export default class FilterTags extends Vue {
-  @Prop() publicTags!: string[];
-  @Prop({ default: () => [] }) tags!: string[];
+  @Prop() publicQualityTags!: types.TagWithProblemCount[];
+  @Prop({ default: () => [] }) tags!: types.TagWithProblemCount[];
   @Prop({ default: () => [] }) selectedTags!: string[];
 
   T = T;
   currentSelectedTags = this.selectedTags;
 
+  get publicQualityTagsText(): string[] {
```

Comment:
nit:

```suggestion
  get publicQualityTagNames(): string[] {
```

acá "texto" es ambiguo porque se puede referir al nombre o a la descripción. mejor se puede usar `name` para ser claro.